packages/clerk-js/src/utils/cookies/handler.ts

```diff
@@ -67,7 +67,7 @@ export const createCookieHandler = () => {
 
   const setDevBrowserCookie = (jwt: string) => {
     const expires = addYears(Date.now(), 1);
-    const sameSite = 'Strict';
+    const sameSite = 'Lax';
```

Comment:
@dimkl do you foresee any issues with e2e testing? My question is related to the changes we made about a month ago